.travis.yml

```diff
@@ -1,5 +1,11 @@
 language: elixir
-elixir: "1.6.4"
+elixir: "1.6.6"
+otp_release: "21.0"
+
+cache:
+  directories:
+  - _build
```

Comment:
Is this not changing with every Travis build anyway?

Reply:
Yes, but we can cache most of it (it means we don't need to compile the deps every-time for example). In most cases, elixir will automatically update the `_build` directory if there are any changes (like local dev). 

From time to time, we may need to clear the cache (if a build is failing when it shouldn't), but mostly this should speed up our builds dramatically.

It also has the added benefit of including the plt for `dialyzer` (stored in `_build` by default).